System/google_chrome_use_cpu.1m.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#  <xbar.title>CPU Google Chrome</xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Alexandre Espinosa Menor</xbar.author>
+#  <xbar.author.github>alexandregz</xbar.author.github>
+#  <xbar.image>https://imgur.com/G7D2XS9</xbar.image>
+#  <xbar.desc>CPU usage from Google Chrome</xbar.desc>
+#  <xbar.dependencies>bash</xbar.dependencies>
+
+APP="Google Chrome.app"
```

Comment:
You could use [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html) for this and the icon. Then, this could become a more generically named `app_use_cpu.1m.sh` plugin with Google Chrome as the default example app on startup.